$NetBSD: patch-aa,v 1.3 2015/12/29 23:34:53 dholland Exp $

Conditionalize math stuff so it can be set from pkgsrc.

--- src/jslibmath.h.orig	2006-07-23 08:22:18.000000000 -0400
+++ src/jslibmath.h
@@ -85,11 +85,7 @@
 
 #ifdef OSSP
 #undef JS_USE_FDLIBM_MATH
-#if defined(__FreeBSD__)
-#define JS_USE_FDLIBM_MATH 0
-#else
-#define JS_USE_FDLIBM_MATH 1
-#endif
+#define JS_USE_FDLIBM_MATH JS_FDLIBM_MATH_REQUESTED
 #endif
 
 #if !JS_USE_FDLIBM_MATH
